src/lib/openapi/spec/edge-token-schema.ts

```diff
@@ -6,19 +6,30 @@ export const edgeTokenSchema = {
     type: 'object',
     additionalProperties: false,
     required: ['token', 'projects', 'type'],
+    description:
+        'A representation of a client token, limiting access to [CLIENT](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens) (used by serverside SDKs) or [FRONTEND](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens) (used by proxy SDKs)',
     properties: {
         projects: {
+            description:
+                'Since a token can have access to multiple projects, but that is represented with a `[]` in the token, this can be used to expand the []',
             type: 'array',
             items: {
                 type: 'string',
             },
+            example: ['developerexperience', 'enterprisegrowth'],
         },
         type: {
+            description: `Unleash supports three different types of API tokens ([ADMIN](https://docs.getunleash.io/reference/api-tokens-and-client-keys#admin-tokens), [CLIENT](https://docs.getunleash.io/reference/api-tokens-and-client-keys#client-tokens), [FRONTEND](https://docs.getunleash.io/reference/api-tokens-and-client-keys#front-end-tokens)). They all have varying access, so when validating a token it's important to know what kind you're dealing with`,
             type: 'string',
             enum: Object.values(ApiTokenType),
+            example: 'client',
         },
         token: {
+            description:
+                '[Unleash API tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys) are comprised of three parts. <project(s)>:<environment>.randomcharacters',
```

Comment:
My question here is "what is a token"? And this is a token schema, right? So if you think of the schema as a variable, then you might have `token.type`, `token.projects`, and ... `token.token`. `token.token` sounds kind of awkward and is confusing. 

We can't change the name of the property, but would you have a different way to describe what this is? We've used `token.secret`, but I don't think that's very illuminating. Because in essence, **this** is the actual token, right? The type and the project access is metadata about the token in a way. 

Something like this?

```suggestion
            description:
                'The actual token value. [Unleash API tokens](https://docs.getunleash.io/reference/api-tokens-and-client-keys) are comprised of three parts. <project(s)>:<environment>.randomcharacters',
```